### DISCUSSION TITLE
Incorrect release dates in microsite

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-Knowlg/Community#89] [Documentation Issues]
@pallakartheekreddy

As per the microsite, the ongoing release 5.3.0 was already released on 16th Jan or Nov '22 but since this is an ongoing release I assume the dates are incorrect. (https://knowlg.sunbird.org/use/release-notes.) Pls correct the dates for 5.3.0 and publish them on the microsite. ED team's cut-off is 3rd Feb to check & integrate the latest prod versions from other BBs.

@vinukumar-vs @gandham-santhosh

**Comment by pallakartheekreddy** [comment]
@Krishnaj20 We have corrected dates for release 5.3.0 and publish on the microsite.
